CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 4.0.1 (Next)
+### 4.0.2 (Next)
 
 * Your contribution here.
 
+### 4.0.1 (4/4/2018)
```

Comment:
Can you please put this back? Rebase against master and leave versioning alone - we bump when we do a release? Thanks!

Reply:
Sure will do :)